.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Feature Requests & Questions 
+    url: https://github.com/sergiodxa/remix-i18next/discussions
+    about: Please ask and answer questions here.
+  - name: 💬 Remix Discord Channel 
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
+    url: https://twitter.com/remix_run
```

Comment:
Remix account will never tweet updates of this library, the lib was made by me not them, and I also don’t tweet about new versions, you should follow releases.

Reply:
Ahh I can see how this might confuse some folks - good point

Reply:
Updated 👌 
- https://github.com/sergiodxa/remix-i18next/pull/41/commits/423a2323dc312918949bd3ef6b1e58da242f5225